Plugin/Source/GUI/ModulatableSlider.cpp

```diff
@@ -12,6 +12,7 @@ void ModulatableSlider::attachToParameter (juce::RangedAudioParameter* param)
 
     attachment = std::make_unique<juce::SliderParameterAttachment> (*param, *this, nullptr);
     modParameter = dynamic_cast<chowdsp::FloatParameter*> (param);
+    modulatedValue = modParameter->getCurrentValue();
```

Comment:
Is it ok to leave modulatedValue without a value until here?

Reply:
Maybe initializing to 0.0 would make sense here? It's a bit tricky since we don't know the parameter range until here.

Reply:
Initializing to 0.0 here instead of `modulatedValue = modParameter->getCurrentValue()`?

Reply:
Oh sorry, I meant initializing to 0.0 in the variable declaration. Initializing like this here makes sense.